index.js

```diff
@@ -161,6 +168,13 @@ class Topic {
     }
   }
 
+  get flattenedChildren(): Array<Topic> {
+    return this.children.reduce((memo, topic) => {
+      memo.push(topic)
+      return memo.concat(topic.flattenedChildren)
+    }, [])
+  }
```

Comment:
`taxonomy.flattenedChildren` is captured in the test as it's being used by `flattenedTopics`

We took a similar approach in testing `taxonomy.selectedTopics` as part of `selectedTopicNames` since it's a utility function and we currently have functional tests only.